modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -801,6 +929,12 @@ private void dropTableLocally(long causalityToken, String name, UUID tblId, List
         }
     }
 
+    private List<ClusterNode> calculateAssignments(TableConfiguration tableCfg, int partNum) {
```

Comment:
Usually it's `partId` rather then `partNum`, or maybe I'm wrong, I don't know

Reply:
Well in this code partId is the id of the partition raft group (its name)